### PR TITLE
Added BellmanFord

### PR DESCRIPTION
**Pull request by Mayank17M**
# Welcome to JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new JavaScript files are placed inside an existing directory.
* [x] All filenames should use the UpperCamelCase (PascalCase) style.  There should be no spaces in filenames.
     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Graphs/BellmanFord.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+//The Bellman–Ford algorithm is an algorithm that computes shortest paths
```
Comment (raklaptudirm) [inline Graphs/BellmanFord.js:1]: Please provide links to the algorithm's description. See https://github.com/TheAlgorithms/Javascript/issues/630 for more details for commenting.

**Review by raklaptudirm — CHANGES_REQUESTED** [review]

File: Graphs/BellmanFord.js (hunk 1 above)
Comment (Mayank17M) [reply Graphs/BellmanFord.js:1]: Sure, I'll make the changes. Thanks!

**Comment by Mayank17M** [comment]
@raklaptudirm Please check if it is appropriate now. Thanks!

**Comment by raklaptudirm** [comment]
Please use `standard.js` to style your code.

**Review by raklaptudirm — APPROVED** [review]
All looks good. I should suggest moving the tests into a separate `jest` file in the `test/` sub directory. You can refer to the other test files if you need so. If you do not feel like doing it, this code is also acceptable, but it will be better if you move the tests.

**Review by raklaptudirm — APPROVED** [review]

**Pull request merged**